code/game/gamemodes/hand_of_god/hand_of_god.dm

```diff
@@ -0,0 +1,390 @@
+		var/datum/team/clockcult/cult = servant.get_team()
+		cult.check_size()
+	
+	if(!silent && L)
+		if(.)
+			to_chat(L, "<span class='heavy_brass'>The world before you suddenly glows a brilliant yellow. [issilicon(L) ? "You cannot compute this truth!" : \
+			"Your mind is racing!"] You hear the whooshing steam and cl[pick("ank", "ink", "unk", "ang")]ing cogs of a billion billion machines, and all at once it comes to you.<br>\
+			Ratvar, the Clockwork Justiciar, [GLOB.ratvar_awakens ? "has been freed from his eternal prison" : "lies in exile, derelict and forgotten in an unseen realm"].</span>")
+			flash_color(L, flash_color = list("#BE8700", "#BE8700", "#BE8700", rgb(0,0,0)), flash_time = 50)
+		else
+			L.visible_message(span_boldwarning("[L] seems to resist an unseen force!"), null, null, 7, L)
+			to_chat(L, "<span class='heavy_brass'>The world before you suddenly glows a brilliant yellow. [issilicon(L) ? "You cannot compute this truth!" : \
+			"Your mind is racing!"] You hear the whooshing steam and cl[pick("ank", "ink", "unk", "ang")]ing cogs of a billion billion machines, and the sound</span> <span class='boldwarning'>\
+			is a meaningless cacophony.</span><br>\
+			<span class='userdanger'>You see an abomination of rusting parts[GLOB.ratvar_awakens ? ", and it is here.<br>It is too late" : \
+			" in an endless grey void.<br>It cannot be allowed to escape"].</span>")
+			L.playsound_local(get_turf(L), 'sound/ambience/antag/clockcultalr.ogg', 40, TRUE, frequency = 100000, pressure_affected = FALSE)
+			flash_color(L, flash_color = list("#BE8700", "#BE8700", "#BE8700", rgb(0,0,0)), flash_time = 5)
+
+/proc/remove_servant_of_ratvar(mob/L, silent = FALSE)
+	if(!L || !L.mind)
+		return
+	var/datum/antagonist/clockcult/clock_datum = L.mind.has_antag_datum(/datum/antagonist/clockcult)
+	if(!clock_datum)
+		return FALSE
+	clock_datum.silent = silent
+	clock_datum.on_removal()
+	return TRUE
+
+//procs bloodcult
+/proc/iscultist(mob/living/M)
+	if(istype(M, /mob/living/carbon/human/dummy))
+		return TRUE
+	return M?.mind?.has_antag_datum(/datum/antagonist/cult)
+
+/proc/is_convertable_to_cult(mob/living/M,datum/team/cult/specific_cult)
+	if(!istype(M))
+		return FALSE
+	if(M.mind)
+		if(ishuman(M) && (M.mind.holy_role))
+			return FALSE
+		if(specific_cult && specific_cult.is_sacrifice_target(M.mind))
+			return FALSE
+		var/mob/living/master = M.mind.enslaved_to?.resolve()
+		if(master && !iscultist(master))
+			return FALSE
+		if(M.mind.unconvertable)
+			return FALSE
+		if(M.is_convert_antag())
+			return FALSE
+	else
+		return FALSE
+	if(HAS_TRAIT(M, TRAIT_MINDSHIELD) || issilicon(M) || isbot(M) || isdrone(M) || ismouse(M) || is_servant_of_ratvar(M) || !M.client)
+		return FALSE //can't convert machines, shielded, braindead, mice, or ratvar's dogs
+	return TRUE
+*/
+/datum/game_mode
+	var/list/hog_clockcult = list() //The Enlightened servants of Ratvar
+	var/list/hog_cult = list() //The devoted cult of Nar'sie
```

Comment:
Do not do this. members should be stored on the mind and not the gamemode. See https://github.com/yogstation13/Yogstation/pull/19524 for better information on it